test/Orleans.Transactions.Tests/Runners/GoldenPathTransactionManagerTestRunner.cs

```diff
@@ -0,0 +1,145 @@
+﻿using System;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+using Orleans.Transactions.Abstractions;
+using System.Diagnostics;
+
+namespace Orleans.Transactions.Tests
+{
+    public class GoldenPathTransactionManagerTestRunner : IDisposable
+    {
+        private readonly TimeSpan logMaintenanceInterval;
+        private readonly TimeSpan storageDelay;
+        private readonly ITestOutputHelper output;
+        private readonly ITransactionManager transactionManager;
+
+        protected GoldenPathTransactionManagerTestRunner(ITransactionManager transactionManager, TimeSpan logMaintenanceInterval, TimeSpan storageDelay, ITestOutputHelper output)
+        {
+            this.transactionManager = transactionManager;
+            this.logMaintenanceInterval = logMaintenanceInterval;
+            this.storageDelay = storageDelay;
+            this.output = output;
+        }
+
+        [SkippableFact]
```

Comment:
if it is `SkippableFact`, then you should have a method in your constructor to determine whether the precondition are met and whether to skip the test. For example, in `AzureStorageBasicTests`, it calls   `TestUtils.CheckForAzureStorage();` to check if azure storage is availale, if not, then throw a `SkipException` which would cause the tests to skip. 
I think you need to do that for at least the `GoldenPathTransactionManagerTest ` since it is using Azure storage?